autoblocks/_impl/testing/run.py

```diff
@@ -256,6 +261,7 @@ async def async_run_test_suite(
         )
 
     await global_state.http_client().post(f"{cli()}/end", json=dict(testExternalId=test_id))
+    current_external_test_id.reset(token)
```

Comment:
this isn't necessary, but more of a safety mechanism

Reply:
Do we want to reset to the previous value? Or just set it to None

Reply:
The previous value would be none here

Reply:
Oh right ok